samples/nrf9160/serial_lte_modem/README.rst

```diff
@@ -109,12 +109,20 @@ The following proprietary BSD socket AT commands are used in this sample:
 * AT#XLISTEN
 * AT#XACCEPT
 * AT#XCONNECT=<url>,<port>
-* AT#XSEND=<data>
+* AT#XSEND=<datatype>,<data>
 * AT#XRECV[=<length>]
-* AT#XSENDTO=<url>,<port>,<data>
+* AT#XSENDTO=<url>,<port>,<datatype>,<data>
 * AT#XRECVFROM=<url>,<port>[,<length>]
 * AT#XGETADDRINFO=<url>
-* AT#XDATATYPE=<type>
+
+Optional TCPIP proxy service
```

Comment:
```suggestion
The following commands are available to control the optional TCP/IP proxy service:
```
Is that correct?

Reply:
These commands are made for a specific customer at the moment and are not available by default, need Config flag to turn them on.

Reply:
Then the documentation is missing information about how to turn these on.

Reply:
Updated with configuration info